src/workerd/io/actor-cache.c++

```diff
@@ -208,8 +208,8 @@ void ActorCache::evictOrOomIfNeeded(Lock& lock) {
 
     // We want to break the OutputGate. We can't quite just do `gate.lockWhile(exception)` because
     // that returns a promise which we'd then have to put somewhere so that we don't immediately
-    // cancel it. Instead, we can ensure that a flush has been scheduled. `flush()`, when called,
```

Comment:
Is all this renaming still useful if the public method is now called `sync()`?

Reply:
Hmmmm, only ever so mildly. We had some functions named `flushImpl*()` and some named `flush*()` and it would be nice to unify them. I can understand if you'd like me to drop that commit now.

Reply:
Nah, it's fine.